key.core/src/main/resources/de/uka/ilkd/key/proof/rules/ruleSetsDeclarations.key

```diff
@@ -288,7 +13,6 @@ The optimized rules pull out select terms before they evaluate them. Because oth
 // 1) NodeInfo#symbolicExecNames
 // 2)
```

Comment:
```suggestion
// 2) OneStepSimplifier#ruleSets
```

Reply:
You are right, that is another place where hard-coded rule sets are used. I also added macro declarations to the list.